src/Appwrite/Specification/Format/OpenAPI3.php

```diff
@@ -396,7 +396,14 @@ public function parse(): array
         foreach ($this->models as $model) {
             foreach ($model->getRules() as $rule) {
                 if (!in_array($rule['type'], ['string', 'integer', 'boolean', 'json', 'float'])) {
-                    $usedModels[] = $rule['type'];
+                    if(\is_array($rule['type'])) {
+                        foreach ($rule['type'] as $key => $value) {
```

Comment:
```suggestion
                        foreach ($rule['type'] as $value) {
```

we never use the `$key` here